channeldb/invoices.go

```diff
@@ -2139,14 +2139,14 @@ func (d *DB) DeleteCanceledInvoices(_ context.Context) error {
 			invoiceIndexBucket,
 		)
 		if invoiceIndex == nil {
-			return invpkg.ErrNoInvoicesCreated
+			return nil
```

Comment:
Could you please also extend the related unit test: https://github.com/lightningnetwork/lnd/blob/e8a1d4876eb6477dbbaf0287e711ce9de56084ee/invoices/invoices_test.go#L2666

Reply:
Pushed a commit modifying the test.